src/ambulance_game/markov/markov.py

```diff
@@ -37,8 +37,13 @@ def build_states(threshold, system_capacity, parking_capacity):
     
     TODO: turn into a generator
     """
+    if parking_capacity < 1:
+        raise ValueError("parking_capacity should be greater or equal to 1")
```

Comment:
Modify message to be in line with other suggestion and add a `TODO` so that we implement being able to do it.